app/models/report_keyword.rb

```diff
@@ -0,0 +1,5 @@
+class ReportKeyword < ApplicationRecord
+  has_and_belongs_to_many :host_reports
+
+  validates :name, uniqueness: true, presence: true
```

Comment:
IMHO uniquness validation is so expensive and there is no added value - if a bug happens and all keywords are added twice for selected reports, is that a big deal? I don't think so. On the other hand, every insert means searching for indices and that is expensive, index is also memory hungry.

Also after we optimize insertion with `upsert_all`, validations will be skipped anyway.

I'd simply drop it on purpose making a comment here that it was intentionally not added.